lib/Controller/DelegationController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @author Cyrille Bollu <cyr.debian@bollu.be> for Arawa (https://www.arawa.fr/)
+ *
```

Comment:
Other files have a `@license` tag. Maybe we should also add it here.

Reply:
Good catch!